src/test/java/stepDefinitions/StepsSignUp.java

```diff
@@ -0,0 +1,125 @@
+package stepDefinitions;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import pages.SignUpPage;
+import utils.FakerManager;
+
+import java.util.Collections;
+
+import static dictionary.ErrorMessages.*;
+
+public class StepsSignUp {
+
+    private SignUpPage singUpPage;
+
+    public StepsSignUp() {
+        this.singUpPage = new SignUpPage();
+    }
+
+    @And("the system shows me the registration form")
+    public void theSystemShowsMeTheRegistrationForm() {
+        Assert.assertEquals("The expected title was not obtained", "Sign Up", singUpPage.getTitleContainerPage());
+    }
+
+    @When("I filled out the fields with valid values")
+    public void iFilledOutTheFieldsWithValidValues() {
+        String userName = FakerManager.getInstance().getFaker().name().username().replace(".", "");
+        String email = FakerManager.getInstance().getFaker().internet().emailAddress();
+        String password = FakerManager.getInstance().getFaker().internet().password();
+        singUpPage.sendTextUserNameField(userName);
+        singUpPage.sendTextEmailField(email);
+        singUpPage.sendTextPasswordField(password);
+    }
+
+    @And("I click the sign up button")
+    public void iClickTheSignUpButton() throws InterruptedException {
+        singUpPage.clickButtonSignUp();
+    }
+
+    @Then("the system displays a successful registration message")
+    public void theSystemDisplaysASuccessfulRegistrationMessage() {
+    }
```

Comment:
This step is empty. Is it what you intended to do?

Reply:
The missing validation is added. As the system does not show a success message when registering, it is validated that the user pic is displayed, after registration